docs/platforms/platforms_index.rst

```diff
@@ -134,6 +134,36 @@ available nodes where possible. ``jsrun`` can also queue runs. However, on
 other cluster and multi-node systems, if auto-resources is disabled, then runs without
 a hostlist or machinefile supplied may be undesirably scheduled to the same nodes.
 
+Zero-resource workers
+~~~~~~~~~~~~~~~~~~~~~
+
+Users with persistent ``gen_f`` functions may notice with certain
+configurations that associated persistent workers are still automatically assigned
+system resources. This can be wasteful if those workers only run in-place and
+don't use the Executor to submit applications to allocated nodes:
+
+.. image:: ../images/persis_wasted_node.png
+    :alt: persis_wasted_node
+    :scale: 40
+    :align: center
+
+This can be resolved within the Executor definition in a calling script. Set the
```

Comment:
in _the_ calling script.

Reply:
This made me think - is calling script in the glossary? I feel like a user could confuse it with the batch script?

Reply:
Good point, calling script currently isn't in the glossary, and should be added and clarified. 

I was also thinking users incorporating libensemble as part of other modules, routines, or studies might prefer not to think of their code as a "libensemble calling script", like how we don't consider our code numpy or psutil calling scripts. But this is just unimportant nitpicking!